RELEASE_PROCESS.md

```diff
@@ -0,0 +1,48 @@
+# Release Processes
```

Comment:
This was renamed, which means we can't see the diff here. Diff can be viewed here: https://github.com/broadinstitute/cromwell/pull/7695/commits/6ad7eddf6bf74431e155f0044423de9c2342d943